internal/queue/loops/control/control.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright (c) 2025 Diagrid Inc.
+Licensed under the MIT License.
+*/
+
+package control
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/dapr/kit/ptr"
+
+	"github.com/diagridio/go-etcd-cron/internal/api/queue"
+	"github.com/diagridio/go-etcd-cron/internal/queue/actioner"
+	"github.com/diagridio/go-etcd-cron/internal/queue/loops"
+)
+
+type Options struct {
+	Actioner actioner.Interface
+	Jobs     loops.Interface[*queue.JobEvent]
+}
+
+type control struct {
+	jobs loops.Interface[*queue.JobEvent]
+	act  actioner.Interface
+}
+
+func New(opts Options) loops.Interface[*queue.ControlEvent] {
+	return loops.New(loops.Options[*queue.ControlEvent]{
+		Handler: &control{
+			jobs: opts.Jobs,
+			act:  opts.Actioner,
+		},
+		BufferSize: ptr.Of(uint64(1024)),
```

Comment:
nit: should this size be a cons?

Reply:
It is different for `counters`  (5) https://github.com/diagridio/go-etcd-cron/pull/79/files#diff-9633f20fe5c8b6230518728488adc6ee3bc7f0ece430d865e23f36fb49234c64R45

This is important because we create many `counters` so we want to reduce the amount of mallocs there.